tcms/management/tests/test_admin.py

```diff
@@ -0,0 +1,37 @@
+from django.urls import reverse
+
+from tcms.tests import LoggedInTestCase
+from tcms.tests.factories import BuildFactory, ComponentFactory
+from tcms.utils.permissions import initiate_user_with_default_setups
+
+
+class TestComponentAdmin(LoggedInTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        initiate_user_with_default_setups(cls.tester)
+        ComponentFactory()
+
+    def test_generated_database_queries(self):
+        self.client.login(  # nosec:B106:hardcoded_password_funcarg
+            username=self.tester.username, password="password"
+        )
```

Comment:
user should already be logged in, after all this is a descendant from LoggedInTestCase

Reply:
I must have overlooked, fixed it. :smile: